src/TextObjects/QuotedString.ts

```diff
@@ -52,6 +56,10 @@ export class TextObjectQuotedString extends TextObject {
 
         while (characterIndex < lineText.length) {
             if (lineText[characterIndex] === this.quoteCharacter) {
+                if(lineText[characterIndex-1] === '\\') {
+                    characterIndex++;
+                    continue;
+                }
```

Comment:
We only appended a condition to previous `if` statement. So how about combine them together?

```ts
if (lineText[characterIndex] === this.quoteCharacter &&
    lineText[characterIndex - 1] !== TextObjectQuotedString.escapeCharacter) {
    // ...
}
```